tests/local_mode/test_doc.py

```diff
@@ -130,3 +130,23 @@ def test_exclude_tokens_on_attr_values_doc():
     # checks if get_vector without excluded_tokens returns a different vector for doc
     # and doc with the word to exclude already missing.
     assert any(doc.get_vector() != doc_excluding_tokens.get_vector())
+
+
+def test_get_token_vectors():
+    """Test the get_token_vectors method"""
+
+    doc = nlp("Joey never ever share food")
+    doc_excluding_tokens = nlp("Joey never share food")
+
+    # add custom_attr to the last token, the word ever
```

Comment:
the word 'ever'

Reply:
It's the difference between the 2 sentences(one exclude ever to test excludedd tokens are ignored)

Reply:
I know, I meant just add the quotes 'ever' instead of ever

Reply:
actually behaviour will be changed due to tokenization

Reply:
my bad, done :)